part-spec/power.json

```diff
@@ -647,7 +647,7 @@
           "description": "definition of each instance of a component in the device",
           "type": "array",
           "items": {
-            "$ref": "definitions.json#/instance"
+            "$ref": "#/instance"
```

Comment:
Yeah I agree this is a little weird. I think moving it into power.json does make sense.

Maybe call this powerComponentDescriptions (or powerComponentDefinitions)

Instances is pretty vague given that I mostly see this for power components for now.

Reply:
Went with powerComponentDefinitions